Makefile

```diff
@@ -25,7 +25,9 @@ endef
 kat: env-if-empty build up
 	@echo
 	@echo "The KAT frontend is running at http://localhost:8000,"
-	@echo "credentials can be found as DJANGO_SUPERUSER_* in the .env file."
+	@echo "An inital superuser has been created: superuser@localhost"
+	@echo "The related password can be found as DJANGO_SUPERUSER_* in the .env file."
+        @echo "run `grep 'DJANGO_SUPERUSER_' .env` to find it."
```

Comment:
Imho the suggested change isn't a big improvement because of two things:
- `superuser@localhost` isn't a fixed value, so you might not want to describe it as such. It might be better to mention that the email/ account name is stored in the `.env` file as `DJANGO_SUPERUSER_EMAIL`.
- `The related password can be found as DJANGO_SUPERUSER_*` could be clearer by specifying `DJANGO_SUPERUSER_PASSWORD` explicitly instead of using the wildcard `*`

I suggest we should just mention that the credentials are stored as `DJANGO_SUPERUSER_EMAIL` and `DJANGO_SUPERUSER_PASSWORD` in the `.env` file. The use of a fixed value like `superuser@localhost` may not be appropriate due to its variability and explicitly mentioning `DJANGO_SUPERUSER_PASSWORD` makes it more clear we're referencing a password variable

Reply:
DJANGO_SUPERUSER_EMAIL. does not exists after running make kat as far as I can see?
Agreed on the other point, unless we do in fact get both vars out of that single command if Email does exists.

Reply:
> DJANGO_SUPERUSER_EMAIL. does not exists after running make kat as far as I can see? Agreed on the other point, unless we do in fact get both vars out of that single command if Email does exists.

`DJANGO_SUPERUSER_EMAIL` exists in the `.env-defaults` file. I'm not sure how the make command currently works, but if you're using the Docker Compose environment it will probably use that variable when building KAT. In any case, `createsuperuser --no-input` will use the environment variables `DJANGO_SUPERUSER_EMAIL` and `DJANGO_SUPERUSER_PASSWORD` to create the superuser account.